SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -142,7 +142,11 @@ void USpatialStatics::PrintTextSpatial(UObject* WorldContextObject, const FText
 
 int64 USpatialStatics::GetActorEntityId(const AActor* Actor)
 {
-	check(Actor);
+	if (Actor == nullptr)
+	{
+		return SpatialConstants::INVALID_ENTITY_ID;
```

Comment:
driveby - can you switch the `return 0` below for SpatialConstants::INVALID_ENTITY_ID also?